tasks/setup-Debian.yml

```diff
@@ -1,4 +1,45 @@
 ---
+- name: Ensure the WebUpd8 repo key is present.
```

Comment:
Instead of adding the WebUpd8 repo in this role, I'd rather have this be a separate role altogether (e.g. like https://github.com/geerlingguy/ansible-role-repo-dotdeb) — that way this role is focused on Java installation, and that role (which can be used for more than just java) is responsible for the repo addition/configuration.

Also, I've never used Oracle's java installation, nor do I plan to... I don't particularly like having to support things like accepting licenses, but an exception might be made as long as the changes to this role are minimal, easy to maintain (consider I will never use that feature), and don't interfere with the main use case (using non-Oracle packages).